File: 47-hard-rna-max-pairs/python/main.py
import json
import sys


def solve(rna, min_loop, allow_wobble):
    pass


def main():
    obj = json.load(sys.stdin)
    print(solve(obj["rna"], obj["min_loop"], obj["allow_wobble"]))


if __name__ == "__main__":
    main()

File: 47-hard-rna-max-pairs/go/main.go
package main

import (
	"encoding/json"
	"fmt"
	"os"
)

type input struct {
	Rna         string `json:"rna"`
	MinLoop     int    `json:"min_loop"`
	AllowWobble bool   `json:"allow_wobble"`
}

func main() {
	var in input
	if err := json.NewDecoder(os.Stdin).Decode(&in); err != nil {
		fmt.Fprintln(os.Stderr, err)
		os.Exit(1)
	}
	fmt.Println(solve(in.Rna, in.MinLoop, in.AllowWobble))
}
